client/components/BarVersion/BarVersion.scss

```diff
@@ -0,0 +1,46 @@
+@import '../../../stylesheets/colors';
+@import '../../../stylesheets/variables';
+
+.bar-graph__bar-version {
+  @include font-size-xs;
+  z-index: 33;
+  font-weight: $font-weight-light;
+  transform: rotate(-180deg);
+  font-variant-numeric: tabular-nums;
+  color: lighten($raven, 15%);
+  transition: opacity 0.2s, color 0.2s;
+  font-family: $font-family-code;
+  letter-spacing: -1px;
+  line-height: 1;
+  cursor: pointer;
+  height: $bar-width;
+  writing-mode: vertical-rl;
+  text-orientation: sideways;
+  height: 50px;
+  text-align: left;
+  justify-content: flex-start;
+  display: inline-block;
+  margin-top: 5px;
+
+  > div {
+    display: flex;
+    text-align: end;
+    justify-content: flex-start;
+    align-items: center;
+    height: 100%;
+    width: 100%;
+    > span {
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
```

Comment:
The ellipsis part doesn't seem to work in Safari.

Reply:
![CleanShot 2022-11-03 at 16 01 52@2x](https://user-images.githubusercontent.com/13868387/199757079-d220a006-719c-4f30-af81-b8e046af6c8f.png)

Looks okay to me?

Reply:
Also here seems to have support https://caniuse.com/?search=ellipsis

Reply:
You need to test with a package with long versions like - `@iconify-prerendered/vue-mdi` to repro.
 If its too hard to get it to work for safari, I wouldn't spend too much time on it.